### ISSUE TITLE
Detect other peer enable/disable camera

### ISSUE THREAD

**Issue by marcoberetta96** [flutter-webrtc/flutter-webrtc#593]
Is it possible to detect when the other user enable or disable the video stream?
The remove RTCVideoView becomes black but I can't figure out a way to detect this event.
It could be done through the data channel, but maybe something about this has already been implemented.
Thanks!

**Comment by Androidsignal** [comment]
@marcoberetta96 Any solution you get ?

**Comment by Androidsignal** [comment]
or any others ?

**Comment by marcoberetta96** [comment]
@Androidsignal  no, still waiting

**Comment by cloudwebrtc** [comment]
Yeah, you need a synchronous signaling mechanism to control it (data-channel or ws message).
